### ISSUE TITLE
Crash when downloading image: `IllegalArgumentException: Scale must be within the range of minScale and maxScale`

### ISSUE THREAD

**Issue by rolandixor** [nextcloud/android#9471]
### Steps to reproduce
1. Download an image

### Expected behaviour
- image downloads

### Actual behaviour
- The client crashes, and the universe collapses into a ball of fire.

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version: 10

Device model: 

Stock or customized system: Near stock (BLU barely customizes their android phones)

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
```

#### Nextcloud log (data/nextcloud.log)
```java
************ CAUSE OF ERROR ************

java.lang.IllegalArgumentException: Scale must be within the range of minScale and maxScale
	at com.github.chrisbanes.photoview.PhotoViewAttacher.setScale(PhotoViewAttacher.java:456)
	at com.github.chrisbanes.photoview.PhotoViewAttacher.setScale(PhotoViewAttacher.java:446)
	at com.github.chrisbanes.photoview.PhotoViewAttacher.setScale(PhotoViewAttacher.java:442)
	at com.github.chrisbanes.photoview.PhotoView.setScale(PhotoView.java:230)
	at com.owncloud.android.ui.preview.PreviewImageFragment.onActivityCreated(PreviewImageFragment.java:213)
	at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:2996)
	at androidx.fragment.app.FragmentStateManager.activityCreated(FragmentStateManager.java:580)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:285)
	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2189)
	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2100)
	at androidx.fragment.app.FragmentManager.execSingleAction(FragmentManager.java:1971)
	at androidx.fragment.app.BackStackRecord.commitNowAllowingStateLoss(BackStackRecord.java:311)
	at androidx.fragment.app.FragmentStatePagerAdapter.finishUpdate(FragmentStatePagerAdapter.java:274)
	at androidx.viewpager.widget.ViewPager.populate(ViewPager.java:1244)
	at androidx.viewpager.widget.ViewPager.populate(ViewPager.java:1092)
	at androidx.viewpager.widget.ViewPager$3.run(ViewPager.java:273)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:967)
	at android.view.Choreographer.doCallbacks(Choreographer.java:791)
	at android.view.Choreographer.doFrame(Choreographer.java:722)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7417)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:980)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30180090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: BLU
Device: G0370WW
Model: G90 PRO
Product: G90_PRO

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 1630729046

```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AlvaroBrey** [comment]
Is the image really big, or really tiny? That's what the stacktrace suggests. Also, what is your app version?

**Comment by rolandixor** [comment]
It was somewhat sizeable, and the app version is 3.18.0.

I should note that the crash didn't repeat when I tried again. I still thought to report the trace just in case it could be useful.

**Comment by joshtrichards** [comment]
Similar to, but not exactly the same as #7029 and #6824.

Hard to reproduce behavior all around. One fix was merged anyhow in #6859, but these issues have been nearly impossible for all reporters to reproduce.

**Comment by joshtrichards** [comment]
Fixed in #13084
